cypress-io/cypress
# feat(launchpad): UNIFY-371 header menu

- Closes https://cypress-io.atlassian.net/browse/UNIFY-376

### Additional details

This PR:

- fixes a hex code in the color tokens
- updates the icon-color-plugin to import our custom colors
- extracts the browser logo imports to a single place so that anywhere that needs them can look them up. This was done in favor of a composable that would link them to graphql after talking with Tim. OpenBrowserList has been refactored to import this.
- adds some icons
- adds a click handler to App.vue that would capture all links with `target="_blank"` and open their href in the default OS browser via graphql mutation, looking for feedback on this idea vs something more explicit (`[target="external"]`?), and some help on the required mutation to get this going.
- adds a Top Nav component and some smaller child components
- adds a transitions folder for reusable transitions since the way of applying this with tailwind's syntax is pretty verbose, but also it's nice to just wrap things in a component to get consistent transitions around the app
- tests the new components via adding a component test for the whole header bar to get graphql mocking working



### How has the user experience changed?
Menus have been added to the header bar in Launch Pad, here's an example with the browser list added for demo purposes:

https://user-images.githubusercontent.com/8340719/135689952-4b1a3311-ad1e-4f6c-84d5-fc4056bc03a3.mov

### How to test

Pull down this branch and run `yarn dev` in the `launchpad` directory. The menu should now exist in the top right of the launchpad. To test the populated browser menu, edit Main.vue to add `:show-browsers="true"` to the tag for `HeaderBar`.

### PR Tasks


- [z] Have tests been added/updated?
